cfgmgr/vlanmgr.cpp

```diff
@@ -362,6 +363,10 @@ void VlanMgr::doVlanTask(Consumer &consumer)
                     mac = fvValue(i);
                     setHostVlanMac(vlan_id, mac);
                 }
+                else if (fvField(i) == "hostif_name")
```

Comment:
Does this mean, for every Vlan thats in VNET, a corresponding host if shall be created?

Reply:
Yes, since it is per VLAN attribute and created for specific VLAN.